microsoft-dynamics-365/2.6/modules/ROOT/pages/microsoft-dynamics-365-connector-reference.adoc

```diff
@@ -0,0 +1,1079 @@
+
+* If a value is specified, set the `-M-Dmule.encoding` VM argument to the same value to avoid an encoding mismatch. | |
+| Config Ref a| ConfigurationProvider |  Name of the configuration used to execute this component. |  | x
+| Target Variable a| String |  Name of the variable that stores the operation's output. |  |
+| Target Value a| String |  Expression that evaluates the operation’s output. The outcome of the expression is stored in the *Target Variable* field. |  `#[payload]` |
+| Reconnection Strategy a| * <<reconnect>>
+* <<reconnect-forever>> |  Retry strategy in case of connectivity errors. |  |
+|===
+
+==== Output
+
+[%autowidth.spread]
+|===
+|Type |<<BulkOperationResult>>
+|===
+
+==== Associated Configurations
+
+* <<dynamics-config>>
+
+==== Throws
+
+* DYNAMICS:CONNECTIVITY
+* DYNAMICS:INVALID_PAGE_SIZE
+* DYNAMICS:NOT_FOUND
+* DYNAMICS:OPERATION_FAILED
+* DYNAMICS:PARSE_ERROR
+* DYNAMICS:RETRY_EXHAUSTED
+* DYNAMICS:TIMEOUT
+* DYNAMICS:UNAUTHORIZED
+* DYNAMICS:UNKNOWN
+
+
+[[delete]]
+=== Delete
+
+`<dynamics:delete>`
+
+Deletes an entity.
+
+* Delete an Entity - `+https://msdn.microsoft.com/en-us/library/mt607664.aspx+`
+
+
+
+
+[%header,cols="20s,25a,30a,15a,10a"]
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | Name of the configuration to use. | | x
+| Logical Name a| String |  Logical name of the entity. It is always the schema name in lower case. |  | x
+| Id a| String |  ID of the entity to delete. |  `#[payload]` |
+| Config Ref a| ConfigurationProvider |  Name of the configuration used to execute this component. |  | x
+| Reconnection Strategy a| * <<reconnect>>
+* <<reconnect-forever>> |  Retry strategy in case of connectivity errors. |  |
+|===
+
+
+==== Associated Configurations
+
+* <<dynamics-config>>
+
+==== Throws
+
+* DYNAMICS:CONNECTIVITY
+* DYNAMICS:INVALID_PAGE_SIZE
+* DYNAMICS:NOT_FOUND
+* DYNAMICS:OPERATION_FAILED
+* DYNAMICS:PARSE_ERROR
+* DYNAMICS:RETRY_EXHAUSTED
+* DYNAMICS:TIMEOUT
+* DYNAMICS:UNAUTHORIZED
+* DYNAMICS:UNKNOWN
+
+
+[[deleteMultiple]]
+=== Delete Multiple
+
+`<dynamics:delete-multiple>`
+
+
+Deletes multiple entities.
+
+* Delete an Entity - `+https://msdn.microsoft.com/en-us/library/mt607664.aspx+`
+* Batch Request - `+https://msdn.microsoft.com/en-us/library/mt607719.aspx+`
+
+
+
+
+[%header,cols="20s,25a,30a,15a,10a"]
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | Name of the configuration to use. | | x
+| Logical Name a| String |  Logical name of the entity. It is always the schema name in lower case. |  | x
+| Ids a| Array of String |  List of IDs to delete. |  `#[payload]` |
+| Use Single Transaction a| Boolean |  Use a single transaction for deletion. If the deletion fails, the transaction is rolled back. |  `true` |
+| Continue On Error a| Boolean |  Continue in case of error. The default value of `false` stops creating records after one create fails.
```

Comment:
Continue in case of error. The default value is to stop deleting records after one delete failed.